Documentation/debugging/using-your-build.md

```diff
@@ -0,0 +1,70 @@
+Using Your Build
+================
+
+This document explains how to use your CoreFX binaries in a .NET Core console app after you've built the repo.
+
+## Overview
+
+You will create a self-contained .NET Core app via `dotnet publish`, which will include stock images of framework assemblies. Then, you will replace some of those assemblies with ones from your CoreFX build.
+
+## Prerequisites
+
+- .NET Core tools - Install at https://www.microsoft.com/net/core
+
+## Steps
+
+0. Create the .NET Core app you want to use with your CoreFX build.
+
+```sh
+mkdir DotnetLab
+cd DotnetLab
+dotnet new
```

Comment:
@mellinoe I wasn't aware that it mattered. I have a `netcoreapp1.1` app I'm following these instructions with, and things are working fine. Copying the files from corefx's `bin/runtime/` folder overwrites virtually every file in the `publish/` directory, including all of the framework assemblies and the runtime DLLs. So if a file produced by `dotnet publish` won't work with netcoreapp2.0, it'll probably be replaced with one from corefx that will.